src/resources/views/filters/select2_multiple.blade.php

```diff
@@ -77,7 +77,14 @@ class="dropdown {{ Request::get($filter->name)?'active':'' }}">
     <script>
 		jQuery(document).ready(function($) {
 			$("select[name=filter_{{ $filter->name }}]").change(function() {
-				var value = JSON.stringify($(this).val());
+                var value = '';
+                if ($(this).val() !== null) {
```

Comment:
And there brobbly should be a new line above this.

Incidentally, I wonder if `is_null` is the right check? Or even `isset` - now I'm nit picking, I know.

Reply:
@lloy0076  Do we really need a new line above `var value`? Seems redundant to me

Reply:
@lloy0076 can you reply to this or update your review? Thanks.